pyServer/manifests/windows/asc-vmhealth

```diff
@@ -1,3 +1,7 @@
 echo,### ASC Extension - Log for VM Health Signals ###
 
 copy,/WindowsAzure/Logs/TransparentInstaller.log
+
+echo,### Registry Hives ###
+copy,/Windows/System32/config/SOFTWARE
```

Comment:
Given that this increases the size of output and it is needed for the new scenario, how about we create a new mode called MinDiagnostic that starts with least amount of information for auto diagnosis.